src/sagemaker/lineage/context.py

```diff
@@ -65,15 +69,15 @@ class Context(_base_types.Record):
     ]
     _boto_delete_members = ["context_name"]
 
-    def save(self):
+    def save(self) -> "Context":
```

Comment:
why are these return types in quotes?

Reply:
Added `__annotations__` from future import and resolved issue. Removed quotes to get pass the issue of 
_NameError: name 'Context' is not defined_

Reply:
Looks like the above is possible from py37 onwards and thus reverting to use quotes